lib/api/_loaders/element-global.js

```diff
@@ -111,7 +128,7 @@ class ElementGlobal {
       abortOnFailure, retryInterval, timeout, suppressNotFoundErrors, index
     } = element;
 
-    if (isDefined(index)) {
+    if (isDefined(index) && typeof(index) == 'number') {
```

Comment:
```suggestion
    if (isDefined(index) && isNumber(index)) {
```
where `isNumber` can be destructed from `Utils` at the top of the file.